modules/cluster-logging-collector-log-forward-kafka.adoc

```diff
@@ -61,8 +61,8 @@ spec:
      labels:
```

Comment:
When the default is  in outputRefs, it is a legacy implementations.  I guess remove the line 60.